swugenerator/generator.py

```diff
@@ -84,10 +84,11 @@ def process_entry(self, entry):
             if "compressed" in entry and not self.nocompress:
                 cmp = entry["compressed"]
                 if cmp not in ("zlib", "zstd"):
-                    logging.critical("Wrong compression algorithm: %s", cmp)
+                    logging.critical("Wrong compression algorithm: \"%s\". Supported only \"zlib\" or \"zstd\".", cmp)
```

Comment:
now you have a duplicated list of supported compression algs, perhaps move that into its own variable

Reply:
I removed this from PR.